frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -225,6 +234,10 @@ public void removeJobQueue(String threadId) {
     }
 
     public boolean addJob(Job job) {
+        if (isUseJobTicket() && !hasJobTickets()) {
```

Comment:
Will a similar check for available job tickets be required in the `public void addJob(String threadId, Job job)` method as well?

Reply:
again hasJobTickets is synchronized func